dist/spec/verifiable-payid-protocol-NC.txt

```diff
@@ -0,0 +1,896 @@
+
+   The Verifiable PayID protocol is designed along the same design
+   principles as [PAYID-PROTOCOL].
+
+3.1.  PaymentInformation Resource as JSON Web Signatures
+
+   We describe the payment accounts(s) information resource per
+   [PAYID-PROTOCOL] that is digitally signed as "PaymentInformation"
+   resource represented as a signed JSON Web Tokens (JWT) [RFC7519]
+   known as JSON Web Signature (JWS) [RFC7515].  JWT is a compact, URL-
+   safe means of representing claims to be transferred between two
+   parties.  JWTs encode claims as a JSON object that is used as the
+   payload of a JWS structure enabling the claims to be digitally
+   signed.  Below we define the three components of JWS.
+
+3.1.1.  JOSE Protected Header
+
+   For JWS, the members of the JSON object represented by the JOSE
+   Header describe the cryptographic operations applied to the JWS
+   Protected header and the JWS payload and optionally additional
+   properties of the JWS.
+
+
+
+Malhotra & Schwartz     Expires January 31, 2021                [Page 4]
+
+Internet-Draft     Verifiable PayID Protocol Extension         July 2020
+
+
+   For a complete list of members of this object, refer to [RFC7519] and
+   [RFC7515].  Following is an example of a JWT object representing JOSE
+   header parameters for JWS JSON Serialization syntax.
+
+   {
+          "name": "identityKey",
+          "alg" : "ES256K",
+          "typ" : "JOSE+JSON",
+          "b64" : "false",
+          "crit": ["b64"],
+          "jwk" :  {
+               "kty": "EC",
+               "use": "sig",
+               "crv": "secp256k1",
+               "x"  : "0",
+               "y". : "0",
+                   },
+   }
+
+3.1.1.1.  name
+
+   The "name" Header Parameter identifies the type of signature.  It is
+   a new OPTIONAL header parameter that is not defined in the IANA JSON
+   Web Signature and Encryption Header Parameters Registry.
+
+3.1.1.2.  alg
+
+   The "alg" (algorithm) Header Parameter identifies the cryptographic
+   algorithm used to secure the JWS.  This is a required field as
+   described in [RFC7515].  We RECOMMEND using "ES256K" which is
+   Elliptic Curve Digital Signature Algorithm (ECDSA) using secp256k1
+   curve-type and SHA-256 hash-type as defined in IANA JSON Web
+   Signature and Encryption Header Parameters Registry.
+
+3.1.1.3.  typ
+
+   The "typ" (type) Header Parameter is used by JWS applications to
+   declare the media type of the complete JWS.  This is an optional
+   field as described in [RFC7515].
+
+3.1.1.4.  b64
+
+   The "b64" (base64url-encode) Header Parameter is an extension to JWS
+   specification that determines how payload is represented in the JWS
+   and the JWS signing input.  When the "b64" value is "false", the
+   payload is represented simply as the JWS Payload value with no
+   encoding; otherwise, it is represented as ASCII(BASE64URL(JWS
+   Payload)).  This is an optional field as described in [RFC7797].
```

Comment:
```suggestion
   Payload).  This is an optional field as described in [RFC7797].
```

Reply:
I don't think the Payload is optional is V.PayID, is it?

Reply:
This sentence refers to parameter "b64" as optional.